package/doc/sphinx/source/documentation_pages/analysis_modules.rst

```diff
@@ -31,6 +31,49 @@ usually follows the same structure
 Please see the individual module documentation for any specific caveats 
 and also read and cite the reference papers associated with these algorithms.
 
+.. rubric:: Using parallelization for built-in analysis runs
+
+Starting from v2.8.0, MDAnalysis ``AnalysisBase`` subclasses can run on a backend
+that supports parallelization (see :mod:`MDAnalysis.analysis.backends`). All analysis
+runs, however, use ``backend='serial'`` by default, which is identical to the previous
+behavior.
+
+Without any dependencies, only one backend is supported -- built-in ``multiprocessing``, that
+processes parts of a trajectory running separate *processes*, i.e. utilizing
+multi-core processors properly. For now, parallelization is added to
+:class:`MDAnalysis.analysis.RMS.RMSD`, but by 3.0 version it will be introduced
+to all subclasses that can support it.
```

Comment:
is it guaranteed that all analyses will be parallelized in 3.0, or is it an aim?

Reply:
It's an aim, although you probably better ask @orbeckst for a better phrasing regarding the future.

Reply:
Looking at the [3.0 roadmap](https://www.mdanalysis.org/2023/10/25/towards_3.0/) we are aiming at parallelization where possible (the roadmap is a bit vague) but not guaranteeing it — @hmacdope do you agree?

I think we can let the current text stand as is. It's a good goal... and will provide plenty of issues where people can try to solve one issue ("parallelize analysis tool X").